server/src/test/java/keywhiz/service/daos/DaoModuleTest.java

```diff
@@ -100,11 +102,13 @@ public void injectsReadwriteSecretContentDAO() {
   }
 
   @Test
-  public void injectsSecretSeriesMapper() {
-    GroupDAO dao = mock(GroupDAO.class);
-    when(groupDAOFactory.readonly()).thenReturn(dao);
+  public void doesNotInjectSecretSeriesMapper() {
+    assertThrows(ConfigurationException.class, () -> getInstance(SecretSeriesMapper.class));
+  }
 
-    assertNotNull(getInstance(SecretSeriesMapper.class));
+  @Test
+  public void injectsSecretSeriesMapperFactory() {
+    assertNotNull(getInstance(SecretSeriesMapper.SecretSeriesMapperFactory.class));
   }
```

Comment:
Because the SecretSeriesMapper must be scoped to a DSLContext we can no longer inject it directly. However, we can directly inject the factory. This gives it similar behavior to existing DAOs.